### ISSUE TITLE
Add Command Line Parameter to avoid accidental drag and drop

### ISSUE THREAD

**Issue by MarcelBochtler** [mPyKen/ScreenAreaShare#23]
I am using `ScreenAreaShare` only from the CLI, because I always want to share the same area in the same size.
The command I'm using is:
```
screen-area-share --maxfps=15 --cx=0 --cy=0 --cw=2560 --ch=1440 --rx=2560 --ry=0 --rh=1440 --freeze
```

Is it possible to add another parameter to avoid accidental drag and drop of the capturing window?

**Comment by mPyKen** [comment]
With the `--freeze` option, the capture window (red border) is hidden and you should not be able to drag it at all. If you mean the rendering window, there is no such option at the moment

**Comment by MarcelBochtler** [comment]
> With the `--freeze` option, the capture window (red border) is hidden and you should not be able to drag it at all.

Yes, the red border is hidden, but I can still drag the capture window when clicking on the very border of the capture window, at least on macOS.
Maybe instead of adding a new parameter, the `--freeze` parameter should avoid dragging the capture window.

**Comment by mPyKen** [comment]
Oh I see - I cannot reproduce this on windows, so it would be a bug for the macos version

**Comment by nnako** [comment]
I see the same effect ON WINDOWS. Even though, the window's own content is "hidden" and transparent, I can click and drag every part of the red border (the mouse icon changes accordingly). As I am working with a large screen, I try to match  each presented application window borders to the red border of the capture window. But if I do, I am forced to grab the red border of the capture window before I have a chance to grab the intended application's window border. As it appears to be "behind" red capture window borders.

As a workaround, I manually drag all the respective application windows in a way that their borders do not match / overlap with the red capture window borders, which causes setup efforts each time I want to present something.

An command line option `--freeze` would help a lot if it causes the borders of the red capture window to appear non-adjustable while facilitating to grab any underlying application window border.

**Comment by mPyKen** [comment]
Hmm that is interesting... Do you use windows 11? It works on my windows 10 machine, i.e. I am unable to drag the capture window when running it with `--freeze` option
